--- TShockAPI/TShockAPI.csproj	2014-07-21 08:08:01.000000000 +1000
+++ TShockAPI/TShockAPI.csproj	2014-12-24 08:11:36.689074288 +1000
@@ -52,6 +52,9 @@
     <PlatformTarget>x86</PlatformTarget>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="TerrariaServerAPI">
+      <HintPath>..\..\build\TerrariaServer.exe</HintPath>
+    </Reference>
     <Reference Include="HttpServer">
       <HintPath>..\HttpBins\HttpServer.dll</HintPath>
     </Reference>
@@ -163,12 +166,6 @@
       <Install>true</Install>
     </BootstrapperPackage>
   </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\TerrariaServerAPI\TerrariaAPI-Server.csproj">
-      <Project>{549A7941-D9C9-4544-BAC8-D9EEEAB4D777}</Project>
-      <Name>TerrariaAPI-Server</Name>
-    </ProjectReference>
-  </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
     <PreBuildEvent>
@@ -190,4 +187,7 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-</Project>
\ No newline at end of file
+  <Target Name="AfterBuild">
+    <Copy SourceFiles="$(TargetPath)" DestinationFolder="$(SolutionDir)\..\build" />
+  </Target>
+</Project>
